src/wp-includes/class-wp-theme-json.php

```diff
@@ -3700,10 +3773,16 @@ public function get_data() {
 	 * Sets the spacingSizes array based on the spacingScale values from theme.json.
 	 *
 	 * @since 6.1.0
+	 * @deprecated 6.6.0
+	 *
+	 * @param string $origin Optional. What source of data to set the spacing sizes for.
+	 *                       One of 'default', 'theme', or 'custom'. Default 'default'.
 	 *
 	 * @return null|void
 	 */
 	public function set_spacing_sizes() {
+		_deprecated_function( __METHOD__, '6.6.0' );
```

Comment:
Is there a `$replacement` method that users should prefer? It can be in the third argument.

https://developer.wordpress.org/reference/functions/_deprecated_function/

Reply:
I think it's fine to follow-up with a fix?

Reply:
There is no replacement. This is being called automatically. That being said, I seem to have missed some comments in the backport that explained that. I'll add them in a follow-up.